general/channels/types/pushinho/tests.py

```diff
@@ -0,0 +1,294 @@
+import os
+
+from django.conf import settings
+from django.urls import reverse
+from django.core.files.uploadedfile import SimpleUploadedFile
+
+from temba.tests import TembaTest
+
+from temba.channels.models import Channel
+from temba.contacts.models import EXTERNAL_SCHEME
+
+from .utils import (
+    MAIN_ICON_URL,
+    CHAT_ICON_URL,
+    upload_icon_to_aws,
+    create_config
+)
+
+from.type import PushinhoType
```

Comment:
Check the import, replace it with the from .type import PushinhoType, a space is missing after the from.